src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectExecContainerResponseHandler.groovy

```diff
@@ -0,0 +1,21 @@
+package com.bmuschko.gradle.docker.response.container
+
+import org.gradle.api.logging.Logger
+import org.gradle.api.logging.Logging
+
+/**
+ * Logs inspect exec container result.
+ *
+ * @see com.bmuschko.gradle.docker.tasks.container.DockerInspectExecContainer
+ */
+class InspectExecContainerResponseHandler {
```

Comment:
I'd actually push back against adding this as I think we want to move away from using this form of callback in favor of the work you did with reactive-streams which IMO is a much better way to go about such things. Thoughts?